requirements/customer/cli.tex

```diff
@@ -0,0 +1,102 @@
+\newcommand{\jacommand}[3]{
+
+\textsc{\large{\textbf{#1}}}
+\begin{itemize}
+\item[\textbf{Options:}] #2
+\item[\textbf{Effect:}] #3
+\end{itemize}
+}
+\newcommand{\jaoption}[3]{
+\begin{itemize}
+\item[] \textbf{#1} (#2)
+\item[\textbf{Effect:}] #3
+\end{itemize}
+}
+\subsection{Command Line Interface}
+All components of JobAdder are accessible through a command line interface (CLI).
+The command line interface on a machine with JobAdder installed has the following structure:
+\begin{equation}
+\text{<component name> <command> <options> <targets>},
+\end{equation}
+where the \textit{component name} can be any one of the following:
+\begin{itemize}
+\item \textbf{jobadder} to access the \textit{user client}.
+\item \textbf{jobcenter} to access the \textit{server}.
+\item \textbf{jobworker} to access the \textit{worker client}.
+\end{itemize}
+Commands, options, and valid targets are and options are specified in the following sections.
+
+Options can be added in the following ways:
+\begin{itemize}
+\item By supplying their names like "--<option\_x> --<option\_y>=2 --<option\_z>".
+\item By supplying their abbreviations like "-x -y 2 -z".
+\item By supplying their abbreviations like "-xz -y 2".
+\end{itemize}
+
+The available \textit{commands} are generally specific to the used component.
+Likewise, the available \textit{options} and \textit{targets} are generally specific to the commands of a used component.
+However, some commands and options are shared between components or commands.
+
+The JobAdder CLI ignores capitalization.
+JobAdder CLI options override the values of configuration files.
+If no command is given, the general help text for the given component is printed out.
+\subsubsection{Shared Options}
+\jaoption{config}{c}{Overrides the global configuration file with the given configuration file.}
```

Comment:
I am not sure how, but we should indicate that the `c`, `d`, etc. are the short versions of the options.

Reply:
I agree.
The formatting of commands and options in the LaTeX document still needs work.